### PR TITLE
Docs: minor update configuration page

### PR DESCRIPTION
**Pull request by kvrhdn**
**What this PR does**:
I noticed we didn't document `http_api_prefix`.

### CODE REVIEW

File: docs/tempo/website/configuration/_index.md [hunk 1]
```diff
@@ -42,12 +42,15 @@ where default_value is the value to use if the environment variable is undefined
...
 
 ## Server
-Tempo uses the Weaveworks/common server. See [here](https://github.com/weaveworks/common/blob/master/server/server.go#L45) for all configuration options.
+Tempo uses the Weaveworks/common server. See [here](https://github.com/weaveworks/common/blob/master/server/server.go#L54) for all configuration options.
 
 ```
 # Optional. Setting to true enables multitenancy and requires X-Scope-OrgID header on all requests.
 [multitenancy_enabled: <bool> | default = false]
 
+# Optional. String prefix for all http api endpoints.
```
Comment (mdisibio) [inline docs/tempo/website/configuration/_index.md:51]:
```suggestion
# Optional. String prefix for all http api endpoints. Must include beginning slash.
```
Should we call out that it must be `/prefix` somehow?

Comment (kvrhdn) [reply docs/tempo/website/configuration/_index.md:51]: Yeah, good call

**Review by mdisibio — APPROVED** [review]

File: docs/tempo/website/configuration/_index.md [hunk 2]
```diff
@@ -42,12 +42,15 @@ where default_value is the value to use if the environment variable is undefined
 You can find more about other supported syntax [here](https://github.com/drone/envsubst/blob/master/readme.md)
 
 ## Server
-Tempo uses the Weaveworks/common server. See [here](https://github.com/weaveworks/common/blob/master/server/server.go#L45) for all configuration options.
+Tempo uses the Weaveworks/common server. See [here](https://github.com/weaveworks/common/blob/master/server/server.go#L54) for all configuration options.
```
Comment (achatterjee-grafana) [inline docs/tempo/website/configuration/_index.md:45]:
```suggestion
Tempo uses the Weaveworks/common server. For more information on configuration options, see [here](https://github.com/weaveworks/common/blob/master/server/server.go#L54).
```

**Review by achatterjee-grafana — COMMENTED** [review]
Minor copy edit suggestion. Otherwsie, lgtm

**Comment by kvrhdn** [comment]
I've applied the suggestion to all similar sections, this should be good to go :+1:

**Review by achatterjee-grafana — APPROVED** [review]

**Pull request merged**